src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTO.kt

```diff
@@ -52,19 +52,19 @@ data class DraftReferralDTO(
       return DraftReferralDTO(
         id = referral.id,
         createdAt = referral.createdAt,
-        completionDeadline = referral.completionDeadline,
+        completionDeadline = referral.referralDetails?.completionDeadline,
         complexityLevels = referral.complexityLevelIds?.ifEmpty { null }
           ?.map { ReferralComplexityLevel(it.key, it.value) }
           ?.sortedBy { it.serviceCategoryId },
-        furtherInformation = referral.furtherInformation,
+        furtherInformation = referral.referralDetails?.furtherInformation,
```

Comment:
I guess the reason why you made the referraleDetails as referralDetails?(nullable) is there can be an instances where the referrralDetails may not be present for a referral?

Reply:
yeh that's right - e.g. in a newly created draft referral